src/modules/comments.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
```

Comment:
- [ ] We setup linters to help us write standardized code. Please enable linters and try fixing the issue involved. If it is a rule that proved hard to solve, you are only allowed to disable that particular rule and not disable linters for the whole file. Even so, code review will still have to figure out whether that rule you are disabling is something that can be easily solved or not.